src/index.html

```diff
@@ -9,6 +9,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class = "header">
+      <a href="#home" class="logo__link">
+        <img
+        class = "logo__img"
+        src="./images/logo.png"
+        alt="...">
```

Comment:
```suggestion
        <img
          class = "logo__img"
          src="./images/logo.png"
          alt="logo"
        >
```
Alt message should be meaningful, not just 3 dots.
In the [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md), you can find useful links where you can read why it's important. 

Also, the checklist:
13. [CODE STYLE] - If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.